students/muhyun/users/urls.py

```diff
@@ -0,0 +1,22 @@
+"""westagram URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.2/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls    import path, include
+from .views import *
+
+urlpatterns = [
+    path('sign_up/', SignUpView.as_view()),
```

Comment:
위코드에서는 최상위 url에서는 /를 붙이지 않고 앱 내 url에서 /로 시작하는 컨벤션을 따르고 있습니다. 수정해주세요!
```suggestion
    path('/sign_up', SignUpView.as_view()),
```